### PR TITLE
Update AboutUs.md and add Kian Seong's PPP

### PR DESCRIPTION
**Pull request by kianseong**
### Changes
- Updated AboutUs.md to include photo.
- Added [@KIANSEONG's Project Portfolio Page ](https://github.com/AY2223S1-CS2103T-T08-2/tp/blob/master/docs/team/kianseong.md) to keep track of contributions.

### CODE REVIEW

File: docs/team/kianseong.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Kian Seong' Project Portfolio Page
```
Comment (beetee17) [inline docs/team/kianseong.md]: You missed a 's here

**Review by beetee17 — COMMENTED** [review]
LGTM apart from a typo.

**Review by wxy1203 — APPROVED** [review]
LGTM

**Review by beetee17 — APPROVED** [review]
LGTM too!

**Pull request merged**